samples/keys/identity_key_usage/src/main.c

```diff
@@ -29,12 +29,6 @@ int main(void)
 	uint8_t key[IDENTITY_KEY_SIZE_BYTES];
 	size_t olen;
 
-	err = nrf_cc3xx_platform_init();
```

Comment:
Why can this removed here without showing up somewhere else?

Reply:
It's here: https://github.com/nrfconnect/sdk-nrf/blob/main/drivers/hw_cc310/hw_cc310.c#L27

Reply:
Ohh ok so it was there for nothing, thanks